packages/extension/src/components/organisms/TokenListing/TokenListing.tsx

```diff
@@ -82,6 +85,13 @@ export const TokenListing: FC<TokenListingProps> = ({ address }) => {
     setExplanationOpen(false);
   }, []);
 
+  const handleFundWallet = useCallback(async () => {
```

Comment:
I don't think you need the `async` as you are not using any `await` later on in the code.

```suggestion
  const handleFundWallet = useCallback(() => {
```